src/style.css

```diff
@@ -0,0 +1,22 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: 50%;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars__star--active {
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+}
```

Comment:
please do not use modificator here
due to checklist you need to use stars--N to dedicate active stars
try to use `:nth-child` 
you can red about it here: https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child